src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/prison/UserService.kt

```diff
@@ -53,11 +55,40 @@ class UserService(
   }
 
   fun findUserByUsername(username: String) = prisonUserApiService.findUserByUsernameWithError(username)
-  fun createLinkedCentralAdminUser(linkUserRequest: CreateLinkedCentralAdminUserRequest) = prisonUserApiService.linkCentralAdminUser(linkUserRequest)
 
-  fun createLinkedLocalAdminUser(linkUserRequest: CreateLinkedLocalAdminUserRequest) = prisonUserApiService.linkLocalAdminUser(linkUserRequest)
+  fun createLinkedCentralAdminUser(linkUserRequest: CreateLinkedCentralAdminUserRequest): PrisonStaffUser {
+    val prisonStaffUser = prisonUserApiService.linkCentralAdminUser(linkUserRequest)
+    sendEmailNotification(linkUserRequest.adminUsername, DPS_ADM, prisonStaffUser)
+    return prisonStaffUser
+  }
+
+  private fun sendEmailNotification(userName: String, userType: UserType, user: PrisonStaffUser) {
+    val createUserRequest = user.primaryEmail?.let {
+      CreateUserRequest(
+        username = userName,
+        email = it,
+        firstName = user.firstName,
+        lastName = user.lastName,
+        userType = userType,
+      )
+    }
+    if (createUserRequest != null) {
+      notificationService.newPrisonUserNotification(createUserRequest, "DPLinkUserCreate")
```

Comment:
Could you move this into the `let` block above i.e. to save the null check on `CreateUserRequest`?

Reply:
Thanks , Updated